allure-framework-modules/mrchecker-webapi-module/src/main/java/com/capgemini/ntc/webapi/pages/httbin/SimpleFormPOSTPage.java

```diff
@@ -0,0 +1,114 @@
+package com.capgemini.ntc.webapi.pages.httbin;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.capgemini.ntc.webapi.core.BasePageWebAPI;
+import com.capgemini.ntc.webapi.core.base.driver.DriverManager;
+import com.capgemini.ntc.webapi.pages.environment.GetEnvironmentParam;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+
+public class SimpleFormPOSTPage extends BasePageWebAPI {
+	
+	private final static String	HOSTNAME	= GetEnvironmentParam.HTTPBIN.getValue();
+	private final static String	PATH		= "/post";
+	private final static String	ENDPOINT	= HOSTNAME + PATH;
+	
+	private static Map<String, Object> requestFields = new HashMap<>();
+	
+	static {
+		requestFields.put("custname", "");
+		requestFields.put("custtel", "");
+		requestFields.put("custemail", "");
+		requestFields.put("comments", "");
+		requestFields.put("delivery", "");
+		requestFields.put("size", "");
+		requestFields.put("topping", "");
+	}
+	
+	public Response submitForm() {
+		return DriverManager.getDriverWebAPI()
```

Comment:
Please check if in getDriverWebAPI()   contentType is not already setup up.

Reply:
Yes true, it's working fine without setting up the *contentType* here.